### PR TITLE
[#726] Platforms thumbnails short description

### LINKED ISSUE #726: Platforms thumbnails should have a friendly name and possibly a short description
The Platform thumbnails (fe. at https://developers.italia.it/it/piattaforme) should include the full project name as well (fe. Anagrafe Nazionale Popolazione Residente (ANPR), Carta d'Identità Elettronica (CIE)) as prescribed by the AAAAA (American Association Against Acronyms Abuse).

A short description below the title would also be a welcome improvement.

image

### PR DESCRIPTION
**Pull request by maurovitale**
## Description

As already mentioned in #726 , this PR tackles:
 - Adds a short description to platforms thumbnails using the software description that were already present in the project
 - Fixs also the layout of the description on mobile devices

## Checklist

- [x] I followed the indications in the [CONTRIBUTING](../CONTRIBUTING.md)
- [ ] The documentation related to the proposed change has been updated accordingly (also comments in code).
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?
- [x] Ready for review! :rocket:

## Fixes

- Fixes #726

### CODE REVIEW

**Comment by bfabio** [comment]
Thanks for the PR @maurovitale. I'd use the normal text color for the subtitle.

**Comment by maurovitale** [comment]
Hi @bfabio Something like this is better ? Let me know so I do the commit. 

Schermata del 2020-10-13 14-05-58

**Comment by bfabio** [comment]
@maurovitale That's perfect :tada:

File: _layouts/platforms.html [hunk 1]
```diff
@@ -30,7 +30,7 @@ <h2 class="subtitle">{{ page.subtitle }}</h2>
                 </figure>
                 {% endif %}
                 <h1 class="projectslist__listing__title">{{platform.title}}</h1>
-                <p>{{platform.subtitle}}</p>
+                <p class="projectslist__listing__subtitle">{{platform.subtitle}}</p>
```
Comment (bfabio) [inline _layouts/platforms.html:33]: We can use the class provided by Bootstrap / Bootstrap italia here.

**Review by bfabio — CHANGES_REQUESTED** [review]

File: _layouts/platforms.html (hunk 1 above)
Comment (maurovitale) [reply _layouts/platforms.html:33]: ah ok, u mean something like this instead ? In this way I could delete the code that I added on the main.scss file.
`<p class="text-dark">{{platform.subtitle}}</p>`

Comment (bfabio) [reply _layouts/platforms.html:33]: Exactly, `text-secondary` should do.

**Review by bfabio — APPROVED** [review]

**Review by sebbalex — APPROVED** [review]
LGTM

**Pull request merged**